### PR TITLE
fix: Token expiration check

### PR DESCRIPTION
**Pull request by peterstarling**
Fixes the issue whereby a new token was generated each time due to an invalid expiration check

### CODE REVIEW

File: packages/squidex/src/auth.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const getAccessTokenFactory = (): (() => Promise<string>) => {
           const jwt = decode<JwtToken>(tk1);
           const currentTime = Date.now() / 1000;
 
-          if (currentTime > jwt.exp) {
+          if (currentTime < jwt.exp) {
```
Comment (kmaid) [inline packages/squidex/src/auth.ts:44]: ooof.

Comment (kmaid) [reply packages/squidex/src/auth.ts:44]: <=?

Comment (niezgoda) [reply packages/squidex/src/auth.ts:44]: Do we really care about this one second? :D

File: packages/squidex/test/auth.test.ts [hunk 2]
```diff
@@ -40,6 +36,57 @@ describe('Get Access Token', () => {
...
+        )
+        .reply(200, {
+          access_token: mockTokenFresh,
+          expires_in: Math.floor(oneHourFromNow.getTime() / 1000),
+          token_type: 'Bearer',
+          scope: 'squidex-api',
+        });
+
+      await getAccessToken();
+      await getAccessToken();
```
[the remark this answers is not in the document]
Comment (peterstarling) [reply packages/squidex/test/auth.test.ts:59]: yeah thats the whole point. Should only call once when invoked twice

**Review by niezgoda — APPROVED** [review]

**Pull request merged**